client/network-gossip/src/state_machine.rs

```diff
@@ -429,6 +454,24 @@ impl<B: BlockT> ConsensusGossip<B> {
 	}
 }
 
+struct Metrics {
+	messages: Gauge<U64>,
+}
+
+impl Metrics {
+	fn register(registry: &Registry) -> Result<Self, PrometheusError> {
+		Ok(Self {
+			messages: register(
+				Gauge::new(
+					"network_gossip_messages_count",
```

Comment:
```suggestion
					"network_gossip_messages_total",
```

In case you use two counters instead of one gauge, the convention is to suffix counters with `_total`. I don't think we need the `_count` suffix though.

See https://prometheus.io/docs/practices/naming/ for more details on the conventions.